libs/blocks/aside/aside.css

```diff
@@ -510,6 +510,88 @@
   gap: var(--spacing-xs);
 }
 
+.aside.promobar.popup {
+  border-radius: var(--spacing-xs);
+  width: var(--grid-container-width);
+  margin: auto;
+  bottom: var(--spacing-xs);
+  box-shadow: 0 3px 6px #707070;
```

Comment:
Is the box shadow always this color. `.dark` & `.light` context? Looks weird to me on a dark bg.

Reply:
@ryanmparrish We have got single box shadow configuration as per the XD and there are no changes in shadow colour based on the background colour theme.